pkg/datastore/database.go

```diff
@@ -36,202 +36,45 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"hash/fnv"
-	"os"
 	"reflect"
-	"strconv"
 	"strings"
 
 	_ "github.com/lib/pq"
 	"github.com/sirupsen/logrus"
 	"github.com/vmware-labs/multi-tenant-persistence-for-saas/pkg/authorizer"
 	"github.com/vmware-labs/multi-tenant-persistence-for-saas/pkg/dbrole"
 	. "github.com/vmware-labs/multi-tenant-persistence-for-saas/pkg/errors"
-	"gorm.io/driver/postgres"
 	"gorm.io/gorm"
 	"gorm.io/gorm/clause"
-	"gorm.io/gorm/logger"
 )
 
 const (
-	// DB Runtime Parameters.
-	DB_CONFIG_ORG_ID = "multitenant.orgId" // Name of Postgres run-time config. parameter that will store current user's org. ID
-
-	// Env. variable names.
-	DB_NAME_ENV_VAR           = "DB_NAME"
-	DB_PORT_ENV_VAR           = "DB_PORT"
-	DB_HOST_ENV_VAR           = "DB_HOST"
-	SSL_MODE_ENV_VAR          = "SSL_MODE"
-	DB_ADMIN_USERNAME_ENV_VAR = "DB_ADMIN_USERNAME"
-	DB_ADMIN_PASSWORD_ENV_VAR = "DB_ADMIN_PASSWORD"
+	DbConfigOrgId               = "multitenant.orgId" // Name of Postgres run-time config. parameter that will store current user's org. ID
+	MAIN          dbrole.DbRole = "main"
```

Comment:
I propose this constant is moved to _dbrole.go_

Reply:
@jeyhun pushed a patch here https://github.com/vmware-labs/multi-tenant-persistence-for-saas/pull/12